### PR TITLE
Moved squad executors data from EndeavourFactory to LabelHandle

### PR DESCRIPTION
**Pull request by semimono**
Also added squad tactics variables to InherentEndeavour GUI.

### CODE REVIEW

File: Assets/Scripts/Attributes/LabelHandle.cs [hunk 1]
```diff
@@ -50,4 +57,34 @@ public class LabelHandle {
...
+
+	public int getConcurrentExecutions(RobotController executor, System.Type endeavourType) {
+		HashSet<RobotController> endeavourExecutors = getExecutors(endeavourType);
+		if (endeavourExecutors.Contains(executor)) {
+			return endeavourExecutors.Count - 1;
+		}
+		return endeavourExecutors.Count;
+	}
+
+	private HashSet<RobotController> getExecutors(System.Type endeavourType) {
```
Comment (michbr) [inline Assets/Scripts/Attributes/LabelHandle.cs]: Never used. What's the purpose of this?

File: Assets/Scripts/Attributes/LabelHandle.cs [hunk 2]
```diff
@@ -50,4 +57,34 @@ public class LabelHandle {
...
+		if (!getAllExecutors().ContainsKey(endeavourType)) {
+			getAllExecutors()[endeavourType] = new HashSet<RobotController>();
+		}
+		return getAllExecutors()[endeavourType];
+	}
+
+	private Dictionary<System.Type, HashSet<RobotController>> getAllExecutors() {
```
Comment (michbr) [inline Assets/Scripts/Attributes/LabelHandle.cs]: Same question. Do we really need this?

Comment (semimono) [inline Assets/Scripts/Attributes/LabelHandle.cs]: We probably don't need either of those. I just copied them over from the original location.

File: Assets/Scripts/Attributes/LabelHandle.cs (hunk 1 above)
Comment (semimono) [inline Assets/Scripts/Attributes/LabelHandle.cs]: It's used in the above 3 methods.

**Pull request merged**